cmd/find.go

```diff
@@ -43,21 +42,17 @@ import (
 
 // findMSG holds JSON and string values for printing
 type findMSG struct {
```

Comment:
I think you have a consistent name like `findMessage`

Reply:
Not part of this PR. Needs a cleanup in separate PR.

Reply:
ok fine